app/app.go

```diff
@@ -94,7 +94,6 @@ func (a *App) InitServer() {
 		if a.Srv().runEssentialJobs {
 			a.Srv().Go(func() {
 				runLicenseExpirationCheckJob(a)
-				runCheckWarnMetricStatusJob(a)
```

Comment:
Does any documentation need to be updated?

Reply:
@catalintomai nice catch! I see a few sections dedicated to Admin Advisor in the docs, most notably this one - https://docs.mattermost.com/administration/notices.html?highlight=admin%20advisor#admin-advisor-notices, though there are a few more - https://docs.mattermost.com/search.html?q=admin+advisor&check_keywords=yes&area=default

@jasonblais should we completely remove those from the docs?

Reply:
Yea, we should update the docs.
- https://docs.mattermost.com/overview/compliance.html#what-information-is-shared-when-i-click-contact-us-on-a-mattermost-admin-advisor-notification can be updated
- https://docs.mattermost.com/administration/notices.html#admin-advisor-notices I would keep and mention they have been paused in v5.35

Reply:
@jasonblais done.